nnoir-onnx/nnoir_onnx/onnx.py

```diff
@@ -231,7 +234,7 @@ def test(self):
     def op_for_node(self, node):
         op_name = f'Op{node.op_type}'
         if op_name in globals():
-            return globals()[op_name](node, self.opset_version)
+            return globals()[op_name](node, self.opset_version, self)
```

Comment:
The conversion of `Split` require shape information of the input.
So I have to pass self to conversion method here.

Reply:
shape information is available from `env`.
see https://github.com/Idein/nnoir/blob/4d86a8210241764b59132b62d77b1ef194a3dc95/nnoir-onnx/nnoir_onnx/operators/average_pool.py#L32-L35

Reply:
thx. I reverted it in dec56b5